### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/boost/commit/2699db461dfa7e219f22eb961056f7321011715c#commitcomment-89062870

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/boost/commit/19643c51ade2a96ba08b53e259cd6f1737a175a2

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: Jamroot [hunk 1]
```diff
@@ -247,7 +247,7 @@ local rule explicit-alias ( id : targets + )
 explicit-alias prg_exec_monitor : libs/test/build//boost_prg_exec_monitor ;
 explicit-alias test_exec_monitor : libs/test/build//boost_test_exec_monitor ;
 explicit-alias unit_test_framework : libs/test/build//boost_unit_test_framework ;
-explicit-alias bgl-vis : libs/graps/build//bgl-vis ;
+explicit-alias bgl-vis : libs/graph/build//bgl-vis ;
```
Comment (jsoref) [inline Jamroot:250]: I can't figure this one out.

File: boostcpp.jam [hunk 2]
```diff
@@ -634,9 +634,9 @@ rule address-model ( )
     return <conditional>@boostcpp.deduce-address-model ;
 }
 
-local deducable-architectures = arm mips1 power riscv s390x sparc x86 combined ;
-feature.feature deduced-architecture : $(deducable-architectures) : propagated optional composite hidden ;
-for a in $(deducable-architectures)
+local deductible-architectures = arm mips1 power riscv s390x sparc x86 combined ;
+feature.feature deduced-architecture : $(deductible-architectures) : propagated optional composite hidden ;
+for a in $(deductible-architectures)
```
Comment (jsoref) [inline boostcpp.jam:639]: Actually, this is probably `deducible`

File: doc/test/gold/document_to_test_formatting/remez.html [hunk 3]
```diff
@@ -221,7 +221,7 @@
     </p>
 <p>
       In single point exchange, we move the control point nearest to the largest
-      extrema to the absissa value of the extrema.
+      extrema to the abscissa value of the extrema.
```
Comment (jsoref) [inline doc/test/gold/document_to_test_formatting/remez.html:224]: ??

File: libs/numeric/sublibs [hunk 4]
```diff
@@ -1 +1 @@
-The existance of this file tells the regression reporting programs that the directory contains sub-directories which are libraries.
\ No newline at end of file
+The existence of this file tells the regression reporting programs that the directory contains sub-directories which are libraries.
```
Comment (jsoref) [inline libs/numeric/sublibs:1]: Note that I try not to interfere with missing newlines-at-eof, although I personally side w/ git/github an wanting them to be present.

File: status/explicit-failures-markup.xml [hunk 5]
```diff
@@ -2246,7 +2246,7 @@
     </library>
 
     <!-- functional/forward -->
-    <library name="functional/foward">
+    <library name="functional/forward">
```
Comment (jsoref) [inline status/explicit-failures-markup.xml:2249]: ?

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

**Review by Flamefire — APPROVED** [review]

**Comment by mclow** [comment]
I have committed these changes (to `develop`) manually, as two commits, to explicitly call out the removal of the line in Jamroot.  Thanks for the patch!

**Pull request closed without merging**